fe/fe-core/src/test/java/com/starrocks/sql/plan/ReplayFromDumpTest.java

```diff
@@ -209,46 +209,47 @@ public void testSSB10() throws Exception {
     public void testTPCDS54() throws Exception {
         Pair<QueryDumpInfo, String> replayPair = getCostPlanFragment(getDumpInfoFromFile("query_dump/tpcds54"));
         // Check the size of the left and right tables
-        Assert.assertTrue(replayPair.second, replayPair.second.contains("  49:NESTLOOP JOIN\n" +
+        Assert.assertTrue(replayPair.second, replayPair.second.contains("54:NESTLOOP JOIN\n" +
```

Comment:
Why this plan changed so much?

Reply:
The plan of TPCDS-Q54 is changed after this pr, but yields right output.

Reply:
is the performance better?

Reply:
No, close in performance of scale 100g.